### ISSUE TITLE
Use TF model object as parameter in new redisai.Model

### ISSUE THREAD

**Issue by CICCIOSGAMINO** [RedisAI/redisai-js#56]
As a Node.js developer i'm using the redisai-js module to connect with Redis-Ai with my node.js application. So i'm using / save / load my models in TensorflowJs format (model.json, weights.bin) so its useful create the Redis-Ai model getting start from this to files saved in my local filesystem. Actually i cannot use codes like:

        const m = await tf.loadLayersModel(`file://model/AX-model/model.json`)
        const myModel = new redisai.Model(redisai.Backend.TF, 'CPU',['a','b'], ['c'], m)
        const r = await aiclient.modelset('mlmodel', myModel)

So i'm interesting to thinking at a solution, in node.js application i'm struggle to use Redis to save the TensorflowJs models too directly in-memory db. I'd like to chose which operation get done in Node.js application and which in Redis-Ai, so for example train the model in Node.js application and predict in Redis-Ai:

         const model = tf.sequential()
         model.add(tf.layers.dense({inputShape: [1], units: 1}))
         model.add(tf.layers.dense({units: 1}))
        model.compile({
               optimizer: 'sgd',   // tf.train.adam(),
               // loss: tf.losses.meanSquaredError,
               loss: 'meanSquaredError',
               metrics: ['mse']
          })

           const myModel = new redisai.Model(redisai.Backend.TF, 'CPU',['a','b'], ['c'], model)
           const r = await aiclient.modelset('mlmodel', myModel)

**Comment by chayim** [comment]
@leibale Want to have a look?
